src/mach/load_command.rs

```diff
@@ -1431,6 +1447,7 @@ pub enum CommandVariant {
     VersionMinWatchos(VersionMinCommand),
     DyldExportsTrie(LinkeditDataCommand),
     DyldChainedFixups(LinkeditDataCommand),
+    Note(NoteCommand),
```

Comment:
I'm realizing this and previous PR of yours are technically breaking changes, glad we got these in now; perhaps would be prudent to add the tag on the enum that it's incomplete (in case a new load command is added and we don't want to add a breaking change again?)